src/activities/imageRecognition/PredictionUpload.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import $ from "jquery";
+import "jquery-ui/ui/widgets/draggable";
+import PropTypes from "prop-types";
```

Comment:
`PropTypes` are built-in to React now, so this can be imported on line 1 like:
```
import React, {PropTypes} from "react";
```

Reply:
I was modeling what we do in the `code-dot-org` repo, but I assume we had them on separate lines from before `PropTypes` were built into React?